sdk/nodejs/runtime/closure/createClosure.ts

```diff
@@ -430,6 +430,54 @@ async function analyzeFunctionInfoAsync(
             throw new Error("Entry for this this function was not created by caller");
         }
 
+        const capturedValues: PropertyMap = new Map();
+
+        // TODO: This does not handle multiple binds correctly.
+        // For example (function () { ... }).bind("a").bind("b") will fail to serialize correctly.
```

Comment:
I think the PR is worth shipping without supporting the multiple bind case. Given the time spent on this, I think it would make sense to wait and see if this is an issue in practice.